README.md

```diff
@@ -211,15 +267,7 @@ For applying Keras models in a distributed manner using Spark, [`KerasImageFileT
 
 The difference in the API from `TFImageTransformer` above stems from the fact that usual Keras workflows have very specific ways to load and resize images that are not part of the TensorFlow Graph.
 
-To use the transformer, we first need to have a Keras model stored as a file. For this notebook we'll just save the Keras built-in InceptionV3 model instead of training one.
-
-
-```python
```

Comment:
I'd keep this because the previous section is very far up and user may not have run it.